SimPEG/electromagnetics/static/utils/static_utils.py

```diff
@@ -356,20 +356,38 @@ def convert_survey_3d_to_2d_lines(
         A DC (or IP) survey
     lineID : (n_data) numpy.ndarray
         Defines the corresponding line ID for each datum
-    data_type : {'volt', 'apparent_resistivity', 'apparent_conductivity', 'apparent_chargeability'}
+    data_type : str, optional
```

Comment:
Leave this as it was, see https://numpydoc.readthedocs.io/en/latest/format.html#parameters. 
"When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:

```
order : {'C', 'F', 'A'}
    Description of `order`.
```

"

Reply:
Good to know. Thanks for the comment!